ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -118,9 +118,9 @@
   "Get the concept-ids from either the concept-ids passed in or 
    from the provider. If both are empty, throws exception." 
   [context concept-ids provider-id]
-  (if-let [concept-ids (if (seq concept-ids)
-                         concept-ids
-                         (get-provider-concept-ids context provider-id))]
+  (if-let [concept-ids (if (= "[\"ALL\"]" (string/trim (string/upper-case concept-ids)))
```

Comment:
I'm not the biggest fan of nested conditionals if there are obvious, idiomatic alternatives, but this is fairly small ... and straight-forward.

Reply:
Thanks. I will see if it looks better to have a separate function.